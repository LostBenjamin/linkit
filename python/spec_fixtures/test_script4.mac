%include include_test2
cyan
magenta
turquoise

