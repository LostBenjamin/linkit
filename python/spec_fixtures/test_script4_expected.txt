line !:line @:line #:line a:line b:line c:line 1:line 2:line 3

